app/screens/HtmlView/__test__/HtmlViewScreen.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { create } from "react-test-renderer";
+
+import HtmlViewScreen from "../HtmlViewScreen";
+
+it("HtmlViewScreen renders correctly", () => {
+  const tree = create(<HtmlViewScreen route />).toJSON();
```

Comment:
`route` is not a boolean

Reply:
yep just haven't made it to actually getting  tests added so that was more of a placeholder for now